File: Python_Contest/areaOfCircleDWITE.js

x1 = 2
y1 = 2
x2 = 4
y2 = 8

r = Math.pow((y2-y1)*(y2-y1)-(x2-x1)*(x2-x1),0.5)

a = 3.14159*r*r
//Rounding

a=a*1000
a = Math.round(a)
a=a/1000

File: Python_Contest/areaOfCircleDWITE.py
import math

def areaOfCircle():
	x1 = input()
	x1 = float(x1)

	y1 = input()
	y1 = float(y1)

	x2 = input()
	x2 = float(x2)

	y2 = input()
	y2 = float(y2)


	r = ((x2-x1)**2 + (y2-y1)**2)**0.5

	a = 3.14159*r*r

	print(a)

areaOfCircle()